Makefile

```diff
@@ -122,12 +122,14 @@ else
   RM_MDEBUG = @:
 endif
 
+GBI_DEFINES := -DF3DEX_GBI_2 -DF3DEX_GBI_PL -DGBI_DOWHILE
```

Comment:
Instead of having `GBI_DEFINES` up here, could you put it with with the other defines (just under `COMMON_DEFINES`) and then add `$(GBI_DEFINES)` to both `AS_DEFINES` and `C_DEFINES`?

Reply:
It's more natural up here since `CC_CHECK_FLAGS` needs it too

Reply:
the `CC_CHECK` compilation also uses `C_DEFINES`, so if you add it there, `CC_CHECK_FLAGS` doesn't need it directly.